theme/common.less

```diff
@@ -1915,33 +1915,8 @@ div.simframe.ui.embed {
         margin-bottom: 20px;
     }
 
-    div.warning {
-        background-color: #F5BC5F;
-        padding: 6px;
-        margin-bottom: 20px;
-    }
-
-    div.container {
-        display: grid;
-        row-gap: 10px;
-        grid-template-rows: 36px auto;
-        grid-template-columns: auto;
-        justify-content: center;
-
-        button {
-            margin: 0;
-        }
-
-        div.prompt {
-            justify-self: center;
-        }
-
-        div.providers {
-            display: grid;
-            row-gap: 20px;
-            grid-template-rows: auto;
-            justify-content: center;
-        }
+    div.remember-me {
```

Comment:
should this be generalize CSS for the actions added to the modal?

Reply:
> should this be generalized CSS for the actions added to the modal?

Good call. Done.